lua/entities/gmod_wire_cd_lock.lua

```diff
@@ -97,7 +97,9 @@ function ENT:AttachDisk(disk)
 
 	self.Const:CallOnRemove("wire_cd_remove_on_weld",function()
 		self.Const = nil
-		self.Disk.Lock = nil
+		if(IsValid(self.Disk)) then
```

Comment:
Redundant parens, also it doesn't need to be valid but w/e

Reply:
Was following the isvalid check on file line 62, which I'd believed to be in the primary style of the rest of the file but it turns out it's just wildly inconsistent